# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/player/tests/data/all.ron
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/player/tests/data/all.ron b/gfx/wgpu/player/tests/data/all.ron
deleted file mode 100644
--- a/gfx/wgpu/player/tests/data/all.ron
+++ /dev/null
@@ -1,11 +0,0 @@
-(
-	backends: (bits: 0xF),
-	tests: [
-		"bind-group.ron",
-		"buffer-copy.ron",
-		"clear-buffer-image.ron",
-		"buffer-zero-init.ron",
-		"pipeline-statistics-query.ron",
-		"quad.ron",
-	],
-)
\ No newline at end of file